### DISCUSSION TITLE
What does it mean to have queriedCount to be undefined?

### DISCUSSION THREAD

**Discussion by awesomelike** [dynamoose/dynamoose#1233] [Q&A]
I am using Query operation as the following:
`Model.query('gsi_hash_key').eq('some_id').using('gsi_name').all();`

After executing it, in the result array, I am getting 1 document and `queriedCount: undefined`. 
I could not find any information on the docs about what it means?

Can anyone help?

**Comment by fishcharlie** [comment]
Sounds like a bug. Please submit an actual issue with an MCVE, and all the requested details.
